### DISCUSSION TITLE
can llama3 work with hierarchical_topics visualization currently?

### DISCUSSION THREAD

**Discussion by keyuchen21** [MaartenGr/BERTopic#2207] [Ideas]
<img width="750" alt="image" src="https://github.com/user-attachments/assets/2ecb7009-ae6c-4220-82de-380bca4263fa">

**Comment by MaartenGr** [comment]
Yes, you can use the `custom_labels` parameter for that. Check out the docstrings of the function for more info.

**Reply by keyuchen21** [reply]
@MaartenGr  Hi, thank you for your help!
would you mind help me take a look where did I do wrong in this case?
<img width="1360" alt="image" src="https://github.com/user-attachments/assets/c4feee1d-d8ed-4922-baf3-f4245fcfdf81" />

**Reply by MaartenGr** [reply]
This seems related to your API key. Perhaps you are not allowed to use that model for whatever reason. You could try [gpt-4o-mini](https://platform.openai.com/docs/models#current-model-aliases) instead which I believe should be both faster and cheaper.

**Reply by semmyk-research** [reply]
@keyuchen21 You probably meant 
`representation_model` and not `rrepresentation_model`
Your client might reference unintended client
